distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/State.enso

```diff
@@ -46,8 +49,9 @@ run key local_state ~computation = @Builtin_Method "State.run"
 
          State.get Float
    ```
-get : Any -> Any ! Uninitialized_State
-get key = @Builtin_Method "State.get"
```

Comment:
- this change is also contributing to goals of
- #6282
- all the `@Builtin_Method` annotations are removed from public `State.enso`
- and moved into `private` `Runtime_Helpers`